### DISCUSSION TITLE
some NIC clients no boot UP

### DISCUSSION THREAD

**Discussion by doguibnu** [ltsp/ltsp#192] [Q&A]
Hello!

We have some clients that no boot UP. @alkisg suggested to do some testing with 2 files about network support boot. The first test was with file undionly.kpxe (/srv/tftp/ltsp) :

picture 01
![01-CYMERA_20191024_090900](https://user-images.githubusercontent.com/38897311/67485580-b6fec300-f640-11e9-98eb-f585e9b4d1a8.jpg)

picture 02
![02-CYMERA_20191024_091139](https://user-images.githubusercontent.com/38897311/67485585-ba924a00-f640-11e9-8c6b-c87cd74e4cf0.jpg)

The second test was replacing file: undionly.kpxe with file ipxe.pxe (/srv/tftp/ltsp)
picture from the same boot client but with file replaced
![drv-2-CYMERA_20191024_093842](https://user-images.githubusercontent.com/38897311/67487171-d9461000-f643-11e9-8bc2-97fe7e107626.jpg)

So, i don't have the knowledge to solve this (if can be solved)

Thanks for attention

Douglas

**Comment by alkisg** [comment]
Douglas, the first two screenshots show an Atheros NIC.
The third one shows both Atheros and Realtek.

Does this PC have two Ethernet cards?
If yes, does it have a hard disk or a floppy, where you could put iPXE on, and connect the cable to the Realtek card?

**Comment by doguibnu** [comment]
Hello @alkisg 

> Douglas, the first two screenshots show an Atheros NIC.
> The third one shows both Atheros and Realtek.
> Does this PC have two Ethernet cards?
No, not, only one on-board NIC  The difference about result screen boot I think that when use first:
undionly.kpxe file
and the the third picture use: 
ipxe.pxe 

> If yes, does it have a hard disk or a floppy, where you could put iPXE on, and connect the cable to the Realtek card?
I can connect other NIC off-board in main board and hard disk to try do it works if you can help or show the procedures......if you have time to this

thanks

**Comment by alkisg** [comment]
The last picture says:

CLIENT MAC ADDR: 00 25 22 89 9B 6F... => This is Atheros

and then a bit later it says:

net0: 00:e0:4c:6a:08:c3 using rtl8169 => This is Realtek

So it clearly reports 2 network cards.
Can you take a photo of the back side of that PC?
